en/docs/learn/rate-limiting/introducing-throttling-use-cases.md

```diff
@@ -56,42 +56,42 @@ Let’s look at how each of these can be important for serving requests through
 
 You can control/restrict access to your API or its selected resources for a given IP address or address range. For example, if you need to grant permission for internal applications to consume a larger quota of your API resource than your external consumers, you can define an advanced policy with higher limits for your internal IP address range and lower limits for the rest. 
 
-Here is a sample for configuring an IP condition by limitting the number of requests for a specific IP address.
+Here is a sample for configuring an IP condition by limiting the number of requests for a specific IP address.
 
-![](../../assets/img/learn/advanced-throttling-ip-condition.png)
+![Advanced Rate Limit policy]({{base_path}}/assets/img/learn/new-allow-specific-ip.png)
 
 ##### HTTP request headers
 
 Advanced policies allow you to apply limits to APIs by filtering requests based on HTTP headers. For example, assume you need to apply a special limit for JSON requests. To do that, you can filter JSON messages by using a policy that inspects the HTTP request headers and checks if the `Content-Type` header is `application/json` and apply a special limit for those requests while allowing a default value for the rest.
 
 Here is a sample for configuring a header condition by considering the "Content-Type" header.
 
-![](../../assets/img/learn/advanced-throttling-header-condition.png)
+![]({{base_path}}/assets/img/learn/new-header-condition-regex.png)
 ##### JWT claims
 
 A JWT claim contains meta information of an API request. It can include application details, API details, user claims, etc. Advanced throttling policies based on JWT claims allow you to filter requests by JWT claim values and apply limits for requests. For example, if you need to allow special limits for users in a specific user role, you can create an advanced policy that checks for a particular regular expression on the role claim of the user and apply special limits for the ones that match.
 
-Here is a sample for configuring JWT claim condition by considering the version of the API (http://wso2.org/claims/version).
+Here is a sample for configuring JWT claim condition by considering the version of the API (http://wso2.org/claims/subscribe).
```

Comment:
Does this look better;
- The following image shows an example for configuring.....

WDYT?

Reply:
Fixed with 8ac6250